hw/xbox/nv2a/gl/gloffscreen.h

```diff
@@ -41,11 +41,13 @@
 #include <GL/gl.h>
 #include <GL/wglext.h>
 #else                                  /* Assume GLX */
+#ifndef __gl_h_                        /* gloffscreen_wgl.c calls them first */
```

Comment:
This feels incredibly hacky.
The proper solution would be to arrange our includes properly or have some custom "gl.h" which handles including everything platform specific in the right order, so we have a functional GL implementation (which is what glo tries to be, I think. So we should fix its internal use of headers).

Reply:
yep, this is horribly hacky :grin: but such hacky patchy patch is a good way to disclose the issue :smiley: 

Well it would raise some concerns about the project if this PR get merged as is without any comment or without any better attempt. :wink:

Reply:
Added `WIP:` to the PR topic to make it more obvious

Reply:
On the other hand, such hacks would be necessary at some point, either here or in an other file.